src/Utils/client.js

```diff
@@ -0,0 +1,72 @@
+import axios from "axios";
+const API_BASE = "https://pokeapi.co/api/v2";
+const POKEMON_TOTAL = 1292; // setting limit for "all pokemon" query
+
+/////////////
+// HELPERS //
+/////////////
+
+/////////////
+// PRUNING // (since we don't want thaaaat much data)
+/////////////
+
+const pruneTypes = (typesData) => {
+  return typesData.map((t) => t.type.name);
```

Comment:
we could potentially have a map of type names to colors to style info boxes, but for now names just seemed like the only relevant data for our purposes

Reply:
that could be a fun feature to add when we finish everything else